src/pizzaiolo/pizzaiolo.service.ts

```diff
@@ -87,7 +87,15 @@ export class PizzaioloService {
   }: PayloadAction) {
     const messageTimeStamp = await this.findMessageTimeStamp({ html_url })
 
-    await slackService.addReaction(icons[review.state], messageTimeStamp)
+    if (review.state === 'approved') {
+      const count = this.findSubmittedPullRequest({ html_url })
+      await slackService.addReaction(
+        icons[`approved_${count}`],
+        messageTimeStamp
+      )
+    } else {
+      await slackService.addReaction(icons[review.state], messageTimeStamp)
+    }
```

Comment:
```suggestion
    if (review.state === 'approved') {
      const count = this.findSubmittedPullRequest({ html_url })
      
      return slackService.addReaction(
        icons[`approved_${count}`],
        messageTimeStamp
      )
      }
    
      return slackService.addReaction(icons[review.state], messageTimeStamp)
```

O que acha mano? Ficou bem-bom essa ideia do count e assim diminui complexidade de leitura, né?